### PR TITLE
Add support for rubrik polaris sdk for python

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Description
Add support for Rubrik Polaris's Python SDK for the integration Rubrik Radar.

### CODE REVIEW

**Comment by crestdatasystems** [comment]
Hey @wissamg, the docker build is failing. Going through the logs, this might be an issue in the agent's environment.  Can you verify that something is not missing from your end?

**Comment by yucohen** [comment]
@crestdatasystems Hi
seems like the problem is because you are not merged from master. Please merge (and resolve your conflicts) and push again, that should solve the issue

File: .dependabot/config.yml.deprecated [hunk 1]
```diff
@@ -2056,14 +2056,28 @@ update_configs:
         dependency_name: demisto/*
         update_type: all
 
+  - package_manager: python
```
Comment (wissamg) [inline .dependabot/config.yml.deprecated:2059]: This file is deprecated and not in use any more please revert the changes.

File: docker/rubrik-polaris-sdk-py3/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM demisto/python3:3.10.1.27636
```
Comment (wissamg) [inline docker/rubrik-polaris-sdk-py3/Dockerfile:1]:
```suggestion
FROM demisto/python3:3.10.4.27798
```

**Review by yucohen — APPROVED** [review]
